tool/travis.sh

```diff
@@ -3,6 +3,8 @@
 # Fast fail the script on failures.
 set -e
 
+(cd tool; git clone https://github.com/flutter/flutter.git)
```

Comment:
do we want to point to alpha, since that's what our users are installing? They will want examples to match their location copy.